tests/dummy/app/templates/collections.hbs

```diff
@@ -0,0 +1,232 @@
+<div class="section index-banner">
+    <div class="container">
+        <div class="row">
+            <div class="col s12 m9">
+                <h1 class="header">Collections</h1>
+            </div>
+        </div>
+    </div>
+</div>
+
+<div class='container'>
+    <div class="section">
+        <h4 class="col s12 header">The component supports the following option:</h4>
+        <div class="intro">
+            <ul>
+                <li>hasHeader, default value <span class="default-value badge">false</span></li>
+            </ul>
+        </div>
+
+        <h4 class="col s12 header">And the single component items supports the following options:</h4>
+        <div class="intro">
+            <ul>
+                <li>isActive, default value <span class="default-value badge">false</span></li>
+                <li>isHeader, default value <span class="default-value badge">false</span></li>
+                <li>isDismissable, default value <span class="default-value badge">false</span></li>
+                <li>linkTo, default value <span class="default-value badge">null</span></li>
+                <li>image, default value <span class="default-value badge">null</span></li>
+                <li>imageAlt, default value <span class="default-value badge">null</span></li>
+                <li>title, default value <span class="default-value badge">null</span></li>
+            </ul>
+        </div>
+    </div>
+
+    <div class="section">
+        <h4 class="col s12 header">Basic</h4>
+
+        <div class="button-example">
```

Comment:
Oh I see, I'll correct it. Btw, also all the other components uses this class... ;-)